src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/Mocks/IPackageRestoreUnconfiguredInputDataSourceFactory.cs

```diff
@@ -33,13 +33,12 @@ public PackageRestoreUnconfiguredDataSource(IReceivableSourceBlock<IProjectVersi
 
             public IComparable DataSourceVersion { get; }
 
-            ISourceBlock<IProjectVersionedValue<object>> IProjectValueDataSource.SourceBlock { get; }
+            ISourceBlock<IProjectVersionedValue<object>> IProjectValueDataSource.SourceBlock => SourceBlock;
```

Comment:
Added implementation here, as it was never being set, which NRT analysis pointed out.